rllab/misc/logger.py

```diff
@@ -185,6 +197,22 @@ def refresh(self):
 
 table_printer = TerminalTablePrinter()
 
+def dump_tensorboard(*args, **kwargs):
+    if len(_tabular)>0 and _tensorboard_writer is not None:
+        tabular_dict = dict(_tabular)
+
+        if 'Iteration' in tabular_dict:
```

Comment:
hmm this is brittle, because we can't guarantee that `'Iteration'` will be logged.

I think the best thing to do right now is to keep our own internal counter which is incremented every time dump_tensorboard() is called. You can also provide a reset_tensorboard_itr() API, in case someone has a different use case

Then we can file an issue about improving the logger, so this isn't so messy.